apps/opencs/view/render/terraintexturemode.cpp

```diff
@@ -328,11 +342,9 @@ void CSVRender::TerrainTextureMode::editTerrainTextureGrid(const WorldspaceHitRe
     std::string mBrushTextureInt = mBrushTexture.substr (hashlocation+1);
     int brushInt = stoi(mBrushTexture.substr (hashlocation+1))+1; // All indices are offset by +1
 
-    int rf = mBrushSize / 2;
-    int r = mBrushSize / 2 + 1;
-    int distance = 0;
+    int r = static_cast<float>(mBrushSize) / 2;
```

Comment:
This whole function is way too long and too repetitive. Major candidate for refactoring. But that is probably best done in a separate PR. Okay for now.

Reply:
@unelsson have you created a separate issue to track this?